groups/sig-k8s-infra/groups.yaml

```diff
@@ -14,7 +14,6 @@ groups:
       ReconcileMembers: "true"
     members:
       - ameukam@gmail.com
-      - amwat@google.com
       - bentheelder@google.com
```

Comment:
we don't really have an emeritus concept here, so instead just noting here for posterity that if amit wishes to continue being in any of these other groups he should be welcome to add himself, I'm only removing these because amwat@google.com is a dangling pointer 😢